### ISSUE TITLE
29_Hack'O'Holicss_PocDoc

### ISSUE THREAD

**Issue by ngdeva99** [CodeBreakMIT/CodeBreak1.0#17]
## ℹ️ Project information
- **Project Name**: PocDoc
- **Short Project Description**: PocDoc, a healthcare application that brings the right care, right at home.
- **Team ID**: 29
- **Team Name**: Hack'O'Holicss
- **Team Members**: 
                 1.  Devanathan NG:  www.github.com/ngdeva99
                2.  Navein Kumar S:  www.github.com/navein19
                 3. Keshav Balachandar:  www.github.com/keshav-b
                 4. Mayank Singh:  www.github.com/Mayanksinghtomar
- **Demo Link**: https://www.youtube.com/watch?v=GxTaozprOzs
- **Repository Link**: PocDoc:  https://github.com/ngdeva99/PocDoc.git
- **Labels**: Deep Learning, Neural Networks,TensorFlow, TFLearn, Android, FireBase, Google MapsPlaces API

### MOTIVATION
- The main motivation behind this project is that, our blood contains humongous data, that we ourselves aren’t aware of. 
- Medical tests and procedures are usually costly and deprived to the needy population of our country. 
- The patient in most cases isn’t aware of what tests are being performed on him/her and these test MIGHT also include medical negligence and inaccuracies. 
- And finally most people aren’t aware on what exactly to do during emergencies.
### OUR IDEA
- To create a universal Pocket Doctor for all kinds of users.
- To make a user friendly medical virtual assistant.
- Locating the nearby Hospitals.
- Analysing Blood Test reports​ & predicting possible diseases.
- Intelligent notification system based on the disease predicted.
- Immediate medical assistance and ambulance service during emergencies

## Technology Stack Used
* Frontend: XML
* Backend: Java, Neural Networks(TFLearn models using python)
* Databases: FireBase
